Mathlib/Algebra/Order/GroupWithZero/WithZero.lean

```diff
@@ -93,11 +92,18 @@ lemma WithZero.withZeroUnitsEquiv_strictMono :
 
 /-- Given any linearly ordered commutative group with zero `α`, this is the order isomorphism
 between `WithZero αˣ` with `α`. -/
-@[simps!]
 def OrderIso.withZeroUnits : WithZero αˣ ≃o α where
   __ := withZeroUnitsEquiv
   map_rel_iff' := WithZero.withZeroUnitsEquiv_strictMono.le_iff_le
 
+
+lemma OrderIso.withZeroUnits_apply (x : WithZero αˣ) :
+    OrderIso.withZeroUnits x = withZeroUnitsEquiv x := rfl
+
+lemma OrderIso.withZeroUnits_mul (x y : WithZero αˣ) :
+    OrderIso.withZeroUnits (x * y) = OrderIso.withZeroUnits x * OrderIso.withZeroUnits y := by
+  simp [OrderIso.withZeroUnits, map_mul]
```

Comment:
```suggestion
lemma OrderIso.withZeroUnits_mul (x y : WithZero αˣ) :
    OrderIso.withZeroUnits (x * y) = OrderIso.withZeroUnits x * OrderIso.withZeroUnits y :=
  withZeroUnitsEquiv.map_mul _ _
```
if you keep it without the simps tag, you could phrase this this way

Reply:
Right, but see my above comments. I don't have strong feelings insofar we can rule out the bad/useless _apply lemma.